app/Models/Campaign.php

```diff
@@ -30,4 +30,20 @@ public function signup()
     {
         return $this->hasMany(Signup::class);
     }
+
+    /**
+     * Get the display start date for the campaign.
+     */
+    public function displayStartDate()
+    {
+        return date("m/d/Y", strtotime($this->start_date));
```

Comment:
Dates should be returned by Eloquent as Carbon objects (assuming they're set as [dates](https://laravel.com/docs/5.7/eloquent-mutators#date-mutators) on the model), so you don't need a separate method to handle this. Instead, in the Blade template:

```blade
<td class="table__cell">{{ $campaign->start_date->format('m/d/Y') }}</td>
<td class="table__cell">{{ $campaign->end_date ? $campaign->end_date('m/d/Y') : '–' }}</td>
```

Reply:
@DFurnes ah I tried that before but didn't work but I think it might be because when I call `format`, it says I can't call that method on a string. However, the `start_date` and `end_date` are both type `DATETIME` in the database - should these be changed to `DATE`? Not sure if that would make a difference though?

Reply:
`DATETIME` and `TIME` are column types in SQL (so how they're persisted in the database, not their type in PHP). Unless you set those fields as [dates](https://laravel.com/docs/5.7/eloquent-mutators#date-mutators) on the model, Eloquent will still provide them (and any other columns) to you as strings regardless of the database schema.

Reply:
OH i see what you're saying now - updated!